File: Architecture/MVC/SimpleMVC_24_04_09/app/src/main/java/com/ini_k/simplemvc_240409/MainActivity.java
package com.ini_k.simplemvc_240409;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button button;
    Person person;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        setVariable();
        setView();
    }
    public void setView(){
        button = (Button) findViewById(R.id.infoBtn);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                person.printInfo(person);
            }
        });
    }
    public void setVariable(){
        person = new Person("김태환", "34");
    }
}

File: Kotiln/SampleAppStudy/Chapter2/app/src/main/java/com/inik/chapter2/MainActivity.kt
package com.inik.chapter2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val numberTextView = findViewById<TextView>(R.id.numberTextView)
        val resetBtn = findViewById<Button>(R.id.resetBtn)
        val plusBtn = findViewById<Button>(R.id.plusBtn)

        var number = 0


        resetBtn.setOnClickListener {
            number =0
            numberTextView.text = number.toString()
            Log.d("OnClick","리셋된 숫자 $number")
        }
        plusBtn.setOnClickListener{
            number += 1
            numberTextView.text = number.toString()
            Log.d("Onclick", "플러스된 숫자 $number")
        }
    }
}